Sdk/src/main/java/com/boxpay/checkout/sdk/BoxPayCheckout.kt

```diff
@@ -34,6 +35,24 @@ class BoxPayCheckout(
         context, token, onPaymentResult, sandboxEnabled, customerShopperToken,isSuccessScreenVisible
     )
 
+    constructor(
+        context: Context,
+        token: String,
+        onPaymentResult: ((PaymentResultObject) -> Unit)?,
+        customerShopperToken: String = "",
+        sandboxEnabled: Boolean = false,
+        isSuccessScreenVisible: Boolean = true,
+        extraParams: HashMap<String, Boolean>
```

Comment:
should be configurationOptions

Reply:
Lets make key as Enum - ConfigurationOptions with following value - 

1. SHOW_UPI_QR_ON_LOAD

Reply:
Lets make it Map instead of HashMap

Reply:
done changes added